README.md

```diff
@@ -65,5 +65,55 @@ that isnt available is annoying. Let alone it be null.
 
 [![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)
 
-
+# PS:Ocelot currently does not support 2.0
+
+## Instruction
+
+1.Configured your Consul first, make sure your Consul machine IP, Port, ServerName.
+
+2.Configure your Key/Value in Consul, Ke name: OcelotConfiguration, the Value for the document shows the Json format: [Value](https://github.com/TomPallister/Ocelot/blob/develop/test/Ocelot.ManualTest/configuration.json)
+
+3.Create a new NetCore console application and apply the Ocelot assembly;
+
+``` 4.Add configuration file configuration. Json (name can be picked up), and the configuration file is:
```

Comment:
This information is here https://github.com/TomPallister/Ocelot/wiki/Service-discovery